main.go

```diff
@@ -98,18 +97,8 @@ func main() {
 	utils.RunSH("dependencies", "curl -L https://github.com/mudler/luet/releases/download/"+*luetVersion+"/luet-"+*luetVersion+"-linux-"+*arch+" --output luet")
 	utils.RunSH("dependencies", "chmod +x luet")
 
-	if *buildx {
-		utils.RunSH("dependencies", "curl -L https://github.com/docker/buildx/releases/download/v0.7.1/buildx-v0.7.1.linux-amd64 --output docker-buildx")
-		utils.RunSH("dependencies", "chmod a+x docker-buildx")
-		utils.RunSH("dependencies", "mkdir -p ~/.docker/cli-plugins")
-		utils.RunSH("dependencies", "mv docker-buildx ~/.docker/cli-plugins")
-		utils.RunSH("dependencies", "docker buildx install")
-		utils.RunSH("dependencies", "docker run --privileged --rm tonistiigi/binfmt --install all")
-	}
-
 	utils.RunSH("dependencies", "mv luet /usr/bin/luet && mkdir -p /etc/luet/repos.conf.d/")
 	utils.RunSH("dependencies", "curl -L https://raw.githubusercontent.com/mocaccinoOS/repository-index/master/packages/luet.yml --output /etc/luet/repos.conf.d/luet.yml")
-	utils.RunSH("dependencies", "luet install -y system/luet")
```

Comment:
shall we put this behind a flag too?

Reply:
but this was always redundant. There is a `luetVersion` flag which already works and installs that luet version. They we come here installing this on top.